deploy/html/editform/editform_row.html

```diff
@@ -11,6 +11,11 @@
 		</label>
 
 		{{ editWidget }}
+
+        {% if boneErrors %}
+            <span>{{ boneErrors[0].errorMessage if boneErrors and (boneErrors[0].severity.value == 3 or boneParams.required) }}</span>
```

Comment:
Can we loop this, to show all messages?
```suggestion
            <span>
                {% for error in boneErrors -%}
                    {{ error.errorMessage if (error.severity.value == 3 or boneParams.required) }}
                    {%- if not loop.last %}<br>{% endif -%}
                {%- endfor %}
            </span>
```